CONTRIBUTING.md

```diff
@@ -114,6 +123,50 @@ Examples of PR titles:
  - test: Add infinity test for `MoveEffect.to`
  - refactor: Optimize the structure of the game loop
 
+
+# Maintainers
+
+These instructions are for the maintainers of Flame.
+
+## Merging a pull request
+
+When merging a pull request, make sure that the title of the merge commit has the correct
```

Comment:
Should we also have an entry about the fact that a PR should not contain multiple fixes/features? We have overlooked that a few times in the past but it does make it difficult to see it all back in the CHANGELOG

Reply:
That's outside of the scope for this section, that should be on the section about making a PR.

Reply:
Since I already touched that section in this PR now I added a note about that too.